lib/src/solver/report.dart

```diff
@@ -130,6 +130,33 @@ class SolveReport {
     }
   }
 
+  /// Displays a two-line message, number of outdated packages and an
+  /// instruction to run `pub outdated` if outdated packages are detected.
+  void reportOutdated() {
+    final outdatedPackagesCount = _result.packages.where((id) {
+      final versions = _result.availableVersions[id.name];
+      // A version is counted:
+      // - if there is a newer version which is not a pre-release and current
+      // version is also not a pre-release
```

Comment:
Nit: final '.' at the end of a sentence

Reply:
Add `or,` since `A version is counted` is continued for the next sentence as well.